webviz_subsurface/plugins/_group_tree/_callbacks.py

```diff
@@ -4,11 +4,11 @@
 import webviz_subsurface_components
 from dash.dependencies import Input, Output, State
 
-from ..group_tree_data import GroupTreeData
+from ._ensemble_group_tree_data import EnsembleGroupTreeData
 
 
-def controllers(
-    app: dash.Dash, get_uuid: Callable, grouptreedata: GroupTreeData
+def plugin_callbacks(
+    app: dash.Dash, get_uuid: Callable, ensemble_dict: Dict[str, EnsembleGroupTreeData]
```

Comment:
Thought: Should naming lean more towards group tree data? As providers always work per ensemble now?

E.g. something like `group_tree_data_dict` or `ensemble_group_tree_data_dict`?